drivers/i2c/i2c_nrfx_twi.c

```diff
@@ -237,43 +237,49 @@ static int twi_nrfx_pm_control(struct device *dev, u32_t ctrl_command,
 }
 #endif /* CONFIG_DEVICE_POWER_MANAGEMENT */
 
+/*
+ * We're only using node labels here because we have per-instance
+ * configuration options like CONFIG_I2C_0_NRF_TWI.
+ *
+ * Those are the only obstacle to moving this to DT_INST APIs.
```

Comment:
Not really. The actual reason is that we need the proper index for `nrfx_twi_##idx##_irq_handler`.

Reply:
I missed that one, thank you.